app/main.py

```diff
@@ -2,0 +1,6 @@
+def copy_file(command: str) -> None:
+    commands = command.split(" ")
+    if commands[1] == commands[2] or len(commands) != 3 or commands[0] != "cp":
```

Comment:
Before accessing `commands[1]` and `commands[2]`, ensure that `commands` has at least 3 elements to avoid potential IndexError. You can rearrange the condition to first check `len(commands) != 3`.